bento_beacon/utils/gohan_utils.py

```diff
@@ -12,7 +12,6 @@
     "referenceBases": "reference",
     "alternateBases": "alternative",
     "assemblyId": "assemblyId",
-    "referenceName": "chromosome",  # could handle accession numbers here, Redmine #1076
```

Comment:
maybe worth noting here in a comment that referenceName is handled separately below?